docs/creating.md

```diff
@@ -136,6 +136,15 @@ variable:
 
 The name of the project root directory.
 
+### `_copier_phase`
+
+The current phase, one of `"prompt"`,`"tasks"`, `"migrate"` or `"render"`.
+
+!!! note
+
+    There is also an additional `"undefined"` phase used when not in any phase, mostly
+    for testing purpose.
```

Comment:
Just a final thought here: I think the `"undefined"` phase isn't mostly used for testing purposes. It's used when rendering, e.g., settings in `copier.yaml` including `exclude` items, `skip_if_exists` items, `subdirectory`. Perhaps we could rephrase this a little?

Reply:
Updated with a more accurate description.